tests/test_diffraction_StaticStructureFactor.py

```diff
@@ -204,8 +221,9 @@ def test_attribute_shapes(self):
             bins, k_max, k_min, num_sampled_k_points
         )
         assert sf.bin_centers.shape == (bins,)
-        assert sf.bin_edges.shape == (bins + 1,)
-        npt.assert_allclose(sf.bounds, (k_min, k_max))
+        if not self.DEBYE:
+            assert sf.bin_edges.shape == (bins + 1,)
+        npt.assert_allclose(sf.bounds, (k_min, k_max), rtol=1e-5, atol=1e-5)
```

Comment:
The Debye method has a small amount of floating point error in the reported `k_min, k_max` after performing the shifting and computing the bin centers (which effectively shifts back to the input values). `k_min` was reporting values around `3e-9` for me, instead of exactly zero. I'm not sure how I would refactor the arithmetic to reduce the floating point error.

Reply:
In that case we can leave it as is.